index.html

```diff
@@ -0,0 +1,175 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Oscafest Summit</title>
+		<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.5.0/font/bootstrap-icons.css">
+		<link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header>
+		<div class="header--social">
+			<ul>
+				<li><a href="#"><i class="bi bi-facebook"></i></a></li>
+        <li><a href="#"><i class="bi bi-twitter"></i></a></li>
+        <li><a href="#">English</a></li>
+        <li><a href="#">My Page</a></li>
+        <li><a href="#">Logout</a></li>
+			</ul>
+		</div>
+		<div class="header--nav--section">
+			<div class="header--hamburger">
+				<img src="images/hamburger.png" alt="Hamburger icon">
+			</div>
+			<a href="/index.html" class="header--logo">
+				<div class="logo--design">
+						<p>OS</p>
+				</div>
+				<p class="logo--text">OS Summit <br>2023</p>
+			</a>
+			<div class="header--nav--container">
+				<nav class="header--nav">
+					<img src="images/Icon.png" class="nav--close--icon" alt="Close Icon">
+					<ul>
+						<li><a href="./about.html">About</a></li>
+						<li><a href="#">Program</a></li>
+						<li><a href="#">Join</a></li>
+						<li><a href="#">Sponsor</a></li>
+						<li><a href="#">News</a></li>
+						<li><a href="#">Campaign</a></li>
+					</ul>
+				</nav>
+			</div>
+		</div>
+		<section class="header--hero--section">
+			<p class="hero--greeting">"Hello! Sharing Wolrd"</p>
+			<h1>creative commons <br>global summit <br>2015</h1>
+			<p class="hero--description">A joyful celebration believing in the value of openness and sharing, creating a positive change with people from over 80 countries is taking place in October, in Korea.
```

Comment:
- [ ] add a little margin right to the headline section as well, and please make the heading font weight bolder, add the font size, so it matches the design
- [ ] note that the text in the rectangle is smaller, while the date below it is bigger.. Please fix that.
- [ ] please personalize the heading and text with that of your subject. Remember, the guidelines state you are to follow the design but personalize the content. this will make your work unique.
yours
![Screenshot (1185)](https://github.com/Akorede3133/oscafest-conference/assets/65298048/2f2da7ac-3d27-42a4-9b05-3195a16290cf)
original
![Screenshot (1186)](https://github.com/Akorede3133/oscafest-conference/assets/65298048/48d37be2-9787-4cec-9a10-d9a51a23a0ad)